### DISCUSSION TITLE
Acme Plugin With Postgresql

### DISCUSSION THREAD

**Discussion by atul161** [Kong/kong#8098] [Help] [answered]
Is there any way to configure acme plugin with Postgresql.
I only see few storage options ie shm, vault etc.

**Comment by fffonion** [comment, marked as answer]
Hi @atul161 , you can use `kong` as storage, it uses the same config storage as Kong (postgresql in your case).

**Reply by atul161** [reply]
@fffonion Ahh Thats Great,

I think there is one more option ie custom configuration of storage using lua.
But  as you told ,postgresql config will  work as a kong config.
Ps:Documentation should be more clear:)
